thunder/torch/__init__.py

```diff
@@ -1786,6 +1786,32 @@ def _reduction(
     return result
 
 
+@torchsymbol(torch.all, is_method=True, id="torch.all")
+def all_tensor(
+    a: TensorLike, /, dim: None | int | Sequence[int] = None, keepdim: bool = False, *, out: None | TensorLike = None
+) -> TensorLike | None:
```

Comment:
The return annotation is `TensorLike | None`, but can this actually return `None`?

Reply:
As `logical_not` does not return a `None`, `all_tensor` cannot return `None`. Let me remove this. Thank you!